doc/releases/changelog-dev.md

```diff
@@ -87,6 +87,16 @@
 * When given a callable, `qml.ctrl` now does its custom pre-processing on all queued operators from the callable.
   [(#4370)](https://github.com/PennyLaneAI/pennylane/pull/4370)
 
+
+* `qml.interfaces.set_shots` accepts `Shots` object as well as `int`'s and tuples of `int`'s.
```

Comment:
should we add the valueerror with new devices to the changelog? i guess in the breaking changes section?

Reply:
Before it was an unhelpful `AttributeError`. So not that much of a change.